Specs/DateTime/Spanish/DatePeriodExtractor.json

```diff
@@ -540,23 +415,27 @@
   },
   {
     "Input": "Estare afuera la tercer semana de 2027",
+    "NotSupported": "dotnet",
+    "NotSupportedByDesign": "javascript, python, java",
     "Results": [
       {
-        "Text": "la tercer semana de 2027",
+        "Text": "tercer semana de 2027",
```

Comment:
In English, this test resolves `the third week of 2027` from the input `I'll be out the third week of 2027` maintaining `the` keyword, so in Spanish, we should keep `la` equivalent.